docs/usage/configuration-options.md

```diff
@@ -159,6 +159,12 @@ It's possible to add this setting into the `renovate.json` file as part of the "
 
 Configuring this to `true` means that Renovate will detect and apply the default reviewers rules to PRs (Bitbucket only).
 
+## branchConcurrentLimit
+
+By default, Renovate won't enforce any concurrent branch limits. If you want the same limit for both concurrent branches and concurrent PRs, then just set a value for `prConcurrentLimit` and it will be reused for branch calculations too. However, if you want to allow more concurrent branches than concurrent PRs, you can configure both values (e.g. `branchConcurrentLimit=5` and `prConcurrentLimit=3`).
```

Comment:
```suggestion
By default, Renovate won't enforce any concurrent branch limits.
If you want the same limit for both concurrent branches and concurrent PRs, then just set a value for prConcurrentLimit` and it will be reused for branch calculations too.
However, if you want to allow more concurrent branches than concurrent PRs, you can configure both values (e.g. `branchConcurrentLimit=5` and `prConcurrentLimit=3`).
```

Suggestion ensures we're following the one sentence per line style.